modules/account-lib/src/coin/ada/index.ts

```diff
@@ -0,0 +1,5 @@
+import * as Utils from './utils';
```

Comment:
i thought we are creating a separate module for ada, i.e. sdk-core-ada, not in account-lib